pkg/machinery/config/types/v1alpha1/generate/generate.go

```diff
@@ -224,7 +255,7 @@ func NewSecretsBundle() (*SecretsBundle, error) {
 }
 
 // NewSecretsBundleFromConfig creates secrets bundle using existing config.
-func NewSecretsBundleFromConfig(c config.Provider) *SecretsBundle {
+func NewSecretsBundleFromConfig(clock Clock, c config.Provider) *SecretsBundle {
```

Comment:
I guess it doesn't need the clock actually here, as it's not going to be used? (or it still generates some certs this way?)

Reply:
clock is used for admin certs creation.